content/blog/2019-06-12-taking-over-maint.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2019-06-12
+slug: taking-over-maint
+title: Taking over maintainence of a software package
+authors:
+  - Scott Chamberlain
+  - Maëlle Salmon
+  - Noam Ross
+categories: blog
+topicid: 
+tags:
+  - software
+  - community
+---
+
+Software is maintained by people. While software can in theory live on indefinitely, to do so requires people. People change jobs, move locations, retire, and unfortuntaley die sometimes. When a software maintainer can no longer maintain a package, what happens to the software?
+
+Because of the fragility of people in software, in an ideal world a piece of software should have as many maintainers as possible. Increasing maintainers increases the so-called [bus factor][bus]. A piece of software should have a high of a number as possible; lower means fewer people have to get hit by a bus to then have no maintainers. 
```

Comment:
```suggestion
Because of the fragility of people in software, in an ideal world a piece of software should have as many maintainers as possible. Increasing maintainers increases the so-called [bus factor][bus]. A lower number of maintainers means fewer people have to get hit by a bus to then have no maintainers. 
```